packages/geoprocessing/src/toolbox/overlapGroupMetrics.ts

```diff
@@ -13,20 +14,61 @@ import {
   isSketchCollection,
   groupBy,
   clip,
+  isPolygonFeatureArray,
 } from "../helpers";
-import { createMetric } from "../metrics";
+import { createMetric, firstMatchingMetric } from "../metrics";
 import { overlapFeatures } from "./overlapFeatures";
 import { overlapArea } from "./overlapArea";
 import flatten from "@turf/flatten";
 import { featureCollection } from "@turf/helpers";
 import cloneDeep from "lodash/cloneDeep";
+import { rasterMetrics } from "./rasterMetrics";
 
 type OverlapGroupOperation = (
   metricId: string,
-  features: Feature<Polygon>[],
+  features: Feature<Polygon>[] | Georaster,
   sc: SketchCollection<Polygon>
 ) => Promise<number>;
 
+/**
+ * Generate overlap group metrics using rasterMetrics operation
+ */
+export async function overlapRasterGroupMetrics(options: {
+  /** Caller-provided metric ID */
+  metricId: string;
+  /** Group identifiers - will generate group metric for each, even if result in zero value, so pre-filter if want to limit */
+  groupIds: string[];
+  /** Sketch - single or collection */
+  sketch: Sketch<Polygon> | SketchCollection<Polygon>;
+  /** Function that given sketch metric and group name, returns true if sketch is in the group, otherwise false */
+  metricToGroup: (sketchMetric: Metric) => string;
+  /** The metrics to group */
+  metrics: Metric[];
+  /** Raster to overlap, keyed by class ID, use empty array if overlapArea operation */
+  featuresByClass: Record<string, Georaster>;
```

Comment:
Should this extra part of comment be dropped? "use empty array if overlapArea operation".  Maybe it was copied over from the other function.

Reply:
Removing that last part, it was an artifact as you expected